auth.py

```diff
@@ -0,0 +1,115 @@
+from datetime import timedelta, datetime
+from typing import Annotated, List
+from fastapi import APIRouter, Depends, HTTPException, status
+from pydantic import BaseModel
+from sqlalchemy.orm import Session
+from models import Users, UserRole
+from passlib.context import CryptContext
+from fastapi.security import OAuth2PasswordRequestForm, OAuth2PasswordBearer
+from jose import jwt, JWTError
+
+from database import get_db
+
+router = APIRouter(
+    prefix='/auth',
+    tags=['auth']
+)
+
+SECRETE_KEY = "ahp9348rh24903hrf23tfg2894tpg34pgh4g4gyes45yh5sej5jjjjts"
```

Comment:
Instead of hard coding the values of SECRETE and ALGORITHM , let keep them in environment variable and access them using os module help.

Change SECRETE_KEY to SECRET_KEY

SECRET_KEY = os.getenv("SECRET_KEY", "your_default_secret_key")